input/weapon-acquisition/event.json

```diff
@@ -11,6 +11,8 @@
     4192,4194,4165,4197,4229,4166,4199,4136,4236,4173,4239,4176,4146,4250,4122,4123,4221,4158,4127,
     // Standalone: Other
     4728,4988,
+    // Aetherized Arsenite / Aetherized Indigo / Aetherized Vermilion
+    11127,11139,11140,
```

Comment:
These are dropped by the strike, but are more reasonably purchased with prophet shards so not sure I would include them here. I generally try to use the event acquisition method only for items that are primarily available from an event rather than other means.

Reply:
The "Event" category no longer exists, it was far too ambiguous, so we replaced it by a "Bound Loot" category quite a while ago, now it's used to track non-tradeable loot, from events or otherwise. Keeping it short, we let players know they can acquire the collectible through loot, so they don't mistake it for a vendor-exclusive item (many such cases).

I don't like renaming files, so that's why it's still called event.json, even if it has nothing to do with events. As for world boss loot, we have the "Other: World Boss" section for individual entries, if that's what you meant.